.github/workflows/validate-packages.yaml

```diff
@@ -0,0 +1,17 @@
+name: Validate package creation
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Build releases
+        run: make builds
+      - name: Creating release packages
+        run: make packages
+      - name: List files
+        run: ls packaging/out/*
```

Comment:
would it be possible to actually assert/expect some files here and fail the action if they are not present?

Reply:
If they are not present the build fails to start with